src/Files.Uwp/App.xaml

```diff
@@ -25,6 +26,10 @@
             <FontFamily x:Key="RecycleBinIcons">/Assets/Custom Glyphs/recyclebin-fabric-glyph.ttf#Fabric MDL2 Assets</FontFamily>
             <FontFamily x:Key="ColoredIconFontFamily">/Assets/Custom Glyphs/Colored-Icons.ttf#Untitled1</FontFamily>
 
+            <fcvt:BooleanToVisibilityConverter x:Key="BooleanToVisibilityConverter" />
+            <fcvt:BooleanToVisibilityInvertConverter x:Key="BooleanToVisibilityInvertConverter" />
+            <fcvt:VisibilityInvertConverter x:Key="VisibilityInvertConverter" />
```

Comment:
Is it posible to use the existing converters?

Reply:
I am not sure I understand you correctly.

I reviewed all existing converters again.
- "BooleanToVisibilityConverter": it's a new one, I can not find any alternative.
- "BooleanToVisibilityInvertConverter": it's redundant, I will remove it. I can use VisibilityInvertConverter, instead.
- "VisibilityInvertConverter": it's an existing one.

If you are saying about the declaration, I can not find any of these converters declared in "TabViewItem_shared.xaml", "HorizontalMultitaskingControl.xaml", these are two files I need to use converters above.

Reply:
Can you declare this on the page where it's being used instead of app.xaml?

Reply:
Yes. I moved converters to the pages using them.